packages/editor-web-component/src/editor-web-component.tsx

```diff
@@ -28,12 +30,12 @@ export class EditorWebComponent extends HTMLElement {
 
```

Comment:
'mode' missing in `observedAttributes` leads to changes in that attribute not being respected. Therefore, only the initial value is respected atm